### ISSUE TITLE
Python error when TERM=tty33

### ISSUE THREAD

**Issue by hughpyle** [xereeto/wordleTTY#1]
With tty33 or tty37 terminal, displays this python error message:
`/usr/local/lib/python3.6/dist-packages/blessed/terminal.py:186: UserWarning: Failed to setupterm(kind='tty33'): setupterm: unknown error`

**Comment by hughpyle** [comment]
Looks like this is just a warning from `blessed`.  I don't think it has any bad side-effects, but does interfere with the display a bit.  So you could avoid it by
```
with warnings.catch_warnings():
   warnings.simplefilter("ignore")
   term = blessed.Terminal()
```

**Comment by xereeto** [comment]
I'll add this but uhh... isn't a tty33 an actual paper teletype? I don't think this will work with one of those since it requires the ability to position the cursor

**Comment by hughpyle** [comment]
Yes, this is true.  It'd not really playable on an original tty33 terminal (for obvious reasons) - I won't mind if you close as "out of scope" ;)

![IMG_8113](https://user-images.githubusercontent.com/150244/153492587-d3160fb7-0593-43e9-afc7-229b1be8a95b.jpg)
![IMG_8109](https://user-images.githubusercontent.com/150244/153492594-c33ca8bc-f95b-4fcd-ad31-329075a490fb.jpg)

**Comment by xereeto** [comment]
Holy shit, this is awesome! I thought I was cool for running it on a vt220... but you win :D 
I'm gonna make a strictly line-based version now... shouldn't take long

**Comment by xereeto** [comment]
OK, it's done and up now... the user `play-lineonly` or `play-lineonly-random` should now work with the TTY!

**Comment by hughpyle** [comment]
great - nice work!

![IMG_8115](https://user-images.githubusercontent.com/150244/153520387-ae342763-7902-4cb9-b549-17c84e8cea48.jpg)

![IMG_8116](https://user-images.githubusercontent.com/150244/153520132-06724bb5-2e07-44d9-bef3-78c514ecf5a6.jpg)

**Comment by xereeto** [comment]
absolutely beyond cool... I need to get myself one of those!

**Comment by xereeto** [comment]
btw... if you're interested in hooking up this marvellous piece of retro tech over a proper modem, i should have that endpoint up and running by tomorrow... :)

**Comment by glitchphan** [comment]
Would you consider updating the repo code with this?
